base/applications/rapps/gui.cpp

```diff
@@ -348,7 +348,7 @@ class CMainToolbar :
             return FALSE;
         }
 
-        ImageList_Destroy((HIMAGELIST) SetImageList(hImageList));
+        ImageList_Destroy((HIMAGELIST)(ULONG_PTR)SetImageList(hImageList));
```

Comment:
CToolbar::SetImageList in rosctrls.h should be fixed to return DWORD_PTR instead